docs/references/nextjs/clerk-middleware.mdx

```diff
@@ -40,12 +46,15 @@ NEXT_PUBLIC_CLERK_AFTER_SIGN_IN_URL=/
 NEXT_PUBLIC_CLERK_AFTER_SIGN_UP_URL=/
 ```
 
-## Protect Multiple Routes
+## Protect routes with `createRouteMatcher()` and `clerkMiddleware()`
```

Comment:
Note that I added "and `clerkMiddleware()` to the end of this heading because the sections that follow [this comment](https://github.com/clerk/clerk-docs/pull/651/files#diff-531eb082fe2e9fa4b5762dfce6bf2c17a5ba9dab679b3e1650a5987a3cbaf972R119) do not involve the `clerkMiddleware()` helper